src/components/Person/_person.scss

```diff
@@ -0,0 +1,116 @@
+// ----------------------------------
+// UI Components
+// Person
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+:root {
+  --person-image-size-small: calc(10 * var(--space-unit)); // 80px
+  --person-image-size-medium: 162px; // temporary
+}
+
+.cc-person {
+  margin-bottom: var(--space-lg);
+
+  @include mq(sm) {
+    display: flex;
+  }
+
+  a {
+    @include animated-link;
+  }
+}
+
+.cc-person__image {
+  @extend %img-cover;
+
+  background-color: var(--colour-grey-70);
+  border-radius: 50%;
+  height: var(--person-image-size-medium);
+  margin-bottom: var(--space-md);
+  overflow: hidden;
+  width: var(--person-image-size-medium);
+
+  @include mq(sm) {
+    margin-bottom: 0;
+    margin-right: calc(6 * var(--space-unit));
+  }
+}
+
+.cc-person__image + .cc-person__wrapper {
+  @include mq(sm) {
+    flex-basis: calc(100% - var(--person-image-size-medium));
+  }
+}
+
+.cc-person__header {
+  @include heading-fancy;
+  margin-bottom: var(--space-md);
+}
+
+.cc-person__heading {
+  @extend %heading-base;
+  @extend %heading-large;
+
+  margin-bottom: 0;
+}
+
+.cc-person__subheading {
+  font-weight: normal;
+}
+
+.cc-person__description {
```

Comment:
I noticed global line-height for Storybook is using an old CC variable name `--body-line-height-md`. It should be updated to `--body-line-height`.

Line height should probably also be set here too since we're doing this for other components.

Reply:
updated :)